app/rankers/algorithms.py

```diff
@@ -130,9 +138,9 @@ def thompson_sampling(
         d = metrics.get(clickdata_id)
         if d:
             # TODO: Decide how many days we want to look back.
-            clicks = max(d.trailing_28_day_opens + alpha_prior, 1e-18)
+            clicks = max(getattr(d, opens_column) + alpha_prior, 1e-18)
             # posterior combines click data with prior (also a beta distribution)
-            no_clicks = max(d.trailing_28_day_impressions - d.trailing_28_day_opens + beta_prior, 1e-18)
+            no_clicks = max(getattr(d, imprs_column) - getattr(d, opens_column) + beta_prior, 1e-18)
```

Comment:
Nits:
- I'd put the `getattr` calls in variables for readability.
- These variables should probably refer to 'opens' instead of 'clicks'.
- I try to avoid abbreviations like imprs because new developers might not know what it means, even if it seems obvious to us. For example, people have to be told that "recs" means "recommendations", even though in my mind they're totally synonymous by now.

Just a suggestion. Please check for accuracy:
```suggestion
            impression_count = getattr(d, impressions_column)
            open_count = getattr(d, opens_column)
            # Compute beta distribution parameters from open and impression counts.
            # Priors provide a boost to explore items, but impression and open counts will dominate the equation as we get more data.
            opens_parameter = max(open_count + alpha_prior, 1e-18)
            no_opens_parameter = max(impression_count - open_count + beta_prior, 1e-18)
```